monetdb/Dockerfile

```diff
@@ -60,6 +62,12 @@ COPY mipengine/udfgen/udfio.py /home/udflib/
 ENV PYTHONPATH "/home/udflib/"
 ENV LOG_LEVEL "INFO"
 
+#######################################################

```

Comment:
1. `Initializing the default users passwords, that will be configured with configure_users.sh`
2. Move it below the default user steps.
3. Add the `guest` user password as env variable.